languages/python/omni_http/src/omni_http/omni_httpd/flask.py

```diff
@@ -27,12 +27,12 @@ class Adapter:
     def __init__(self, app: flask.Flask):
         self.app = app
 
-    def __call__(self, req: HTTPRequest) -> HTTPResponse:
+    def __call__(self, req: HTTPRequest) -> HTTPOutcome:
         res = responder()
         response = self.app.wsgi_app(req.wsgi_environ(), res)
 
         body = b""
         for chunk in response:
             body += chunk
 
-        return HTTPResponse(status = res.status, body = body, headers = res.headers)
\ No newline at end of file
+        return HTTPResponse(status = res.status, body = body, headers = res.headers).outcome()
```

Comment:
Should this be `HTTPResponse` for the testability we discussed?

Reply:
The test is still there in a way. This was reverted given your suggestion here - https://github.com/omnigres/omnigres/pull/316#discussion_r1376973487

Reply:
I may not have been 100% clear there. I meant perhaps we should look into making omni_httpd server accept http_response in addition to http_outcome. This way, Python stuff can return http response without any "outcomes" if it doesn't need to do things like reproxying.